Test/AwsCommonRuntimeKitTests/mqtt/Mqtt5ClientTests.swift

```diff
@@ -904,27 +995,267 @@ class Mqtt5ClientTests: XCBaseTestCase {
             XCTAssertNotEqual(keepAliveInterval, negotiatedSettings.serverKeepAlive)
         } else {
             XCTFail("Missing negotiated settings")
+            return
         }
 
         try client.stop()
         if testContext.semaphoreDisconnection.wait(timeout: .now() + 5) == .timedOut {
             print("Disconnection timed out after 5 seconds")
             XCTFail("Disconnection timed out")
+            return
         }
 
         if testContext.semaphoreStopped.wait(timeout: .now() + 5) == .timedOut {
             print("Stop timed out after 5 seconds")
             XCTFail("Stop timed out")
+            return
         }
     }
 
-    /** Operation Tests [OP-UC] */
+    /*===============================================================
+                     OPERATION TESTS
+    =================================================================*/
+    /*
+    * [Op-UC1] Sub-Unsub happy path
+    */
+    func testMqtt5SubUnsub() async throws {
+        try skipIfPlatformDoesntSupportTLS()
+        let inputHost = try getEnvironmentVarOrSkipTest(environmentVarName: "AWS_TEST_MQTT5_IOT_CORE_HOST")
+        let inputCert = try getEnvironmentVarOrSkipTest(environmentVarName: "AWS_TEST_MQTT5_IOT_CORE_RSA_CERT")
+        let inputKey = try getEnvironmentVarOrSkipTest(environmentVarName: "AWS_TEST_MQTT5_IOT_CORE_RSA_KEY")
+
+        let tlsOptions = try TLSContextOptions.makeMTLS(
+            certificatePath: inputCert,
+            privateKeyPath: inputKey
+        )
+        let tlsContext = try TLSContext(options: tlsOptions, mode: .client)
+
+        let clientOptions = MqttClientOptions(
+            hostName: inputHost,
+            port: UInt32(8883),
+            tlsCtx: tlsContext)
+
+        let testContext = MqttTestContext()
+        let client = try createClient(clientOptions: clientOptions, testContext: testContext)
 
-    // Sub Happy Path
-    func testSubscription() async throws {
+        try client.start()
+        if testContext.semaphoreConnectionSuccess.wait(timeout: .now() + 5) == .timedOut {
+            print("Connection Success Timed out after 5 seconds")
+            XCTFail("Connection Timed Out")
+            return
+        }
+
+        let topic = "test/MQTT5_Binding_Swift_" + UUID().uuidString
+        let subscribePacket = SubscribePacket(topicFilter: topic, qos: QoS.atLeastOnce, noLocal: false)
+
+        let subackPacket: SubackPacket =
+            try await withTimeout(client: client, seconds: 2, operation: {
+                try await client.subscribe(subscribePacket: subscribePacket)
+            })
+        print("SubackPacket received with result \(subackPacket.reasonCodes[0])")
```

Comment:
Are prints normal in tests?

Reply:
Only if you're an eventstreamrpc test >,<